### PR TITLE
Fix node distribution

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

Fix node distribution

### Changes

- check if data is returned before displaying it
- adding note explaining which networks are used in the map

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/189

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings

### CODE REVIEW

File: src/utils/fetchData.ts [hunk 1]
```diff
@@ -41,21 +48,31 @@ function fetchStats(url: string) {
...
+export let status = writable <IStatus>({
+  dev: false,
+  test: false,
+  main: false
+}) 
+
 export async function fetchData() {
 
-  // console.log(await fetchStatsDev()) ;
   const dev = await fetchStats("https://gridproxy.dev.grid.tf/stats?status=up")();
```
Comment (xmonader) [inline src/utils/fetchData.ts:58]: can't any of these raise an exception?

File: src/utils/fetchData.ts [hunk 2]
```diff
@@ -41,21 +48,35 @@ function fetchStats(url: string) {
...
+      ...merge(dev || {}, test || {}, main || {}),
+      nodesDistribution: merge(
+        dev?.nodesDistribution || {},
+        test?.nodesDistribution || {},
+        main?.nodesDistribution || {}
+      ),
+    } as IStatsRes;
+  } 
+  catch (error) {
+    throw new Error(error)
```
Comment (AhmedHanafy725) [inline src/utils/fetchData.ts]: what is the point of the try-catch block if it will throw the error at the end without handling the error?

Comment (zaelgohary) [reply src/utils/fetchData.ts]: I removed the try catch block as there is a catch promise in [fatchData](https://github.com/threefoldtech/tfgrid_stats/blob/69b83f0abe700c43f3af7b9705fa89c7bd0b7600/src/utils/fetchData.ts#L42)

File: src/utils/fetchData.ts (hunk 1 above)
Comment (zaelgohary) [reply src/utils/fetchData.ts:58]: This is already handled in App.vue [here](https://github.com/threefoldtech/tfgrid_stats/blob/83569c43986001cad7147fb9f5366cdfaf467e46/src/App.svelte#L15-L23) & [here](https://github.com/threefoldtech/tfgrid_stats/blob/83569c43986001cad7147fb9f5366cdfaf467e46/src/App.svelte#L82)

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**